api-gateway/src/main/resources/static/scripts/app.js

```diff
@@ -9,7 +9,8 @@ const whiteList = new Set([
 var petClinicApp = angular.module('petClinicApp', [
     'ui.router', 'layoutNav', 'layoutFooter', 'layoutWelcome', 'ownerList', 'ownerDetails', 'ownerForm', 'petForm'
     , 'visits', 'vetList','vetForm','vetDetails', 'loginForm', 'rolesDetails', 'signupForm', 'billDetails', 'billsByOwnerId', 'billHistory'
-    , 'verification' , 'adminPanel']);
+    , 'verification' , 'adminPanel' , 'verification' , 'adminPanel', ]);
```

Comment:
code is duplicated and visitList & billForm are missing (removed accidentally in previous commit)